tests/rules/assertions/objectTypeCurlyNewline.js

```diff
@@ -0,0 +1,93 @@
+export default {
+  invalid: [
+    // Never
+    {
```

Comment:
It would be good to have some tests when there object has multiple properties

Reply:
Yes I can, just to note though this rule does not cater to multi-lining multiple properties, that'll be handled by a separate rule hopefully in the near future just like standard objects https://eslint.org/docs/rules/object-property-newline